Documentation/ReleaseNotes/NuttX-10.1.0

```diff
@@ -140,7 +140,7 @@ NuttX-10.1.0
 * [#2107](https://github.com/apache/nuttx/pull/2107) build: Makefile should reference CONFIG_HOST_xxx
 * [#2392](https://github.com/apache/nuttx/pull/2392) configure.sh: support custom board build config with absolute path
 * [#2824](https://github.com/apache/nuttx/pull/2824) build/esp32: Create Flash size configuration and a new DOWNLOAD make target
-* [#3317](https://github.com/apache/nuttx/pull/3317) tols: Update python scripts to py3 and lint all of them againt Black
+* [#3317](https://github.com/apache/nuttx/pull/3317) tols: Update python scripts to py3 and lint all of them against Black
```

Comment:
should we avoid change the commit message which is part of the unfixable git history.